src/test/java/com/ebivariation/contigalias/NCBIBrowserTest.java

```diff
@@ -54,4 +55,10 @@ void getGenomeReportDirectoryGCFTest() throws IOException {
         assertEquals("GCF/007/608/995/GCF_007608995.1_ASM760899v1/", path);
     }
 
+    @Test
+    void getAssemblyReportInputStream() throws IOException {
+        InputStream stream = ncbiBrowser.getAssemblyReportInputStream("/genomes/all/GCF/007/608/995/GCF_007608995.1_ASM760899v1/");
+        assertTrue(stream.read() != -1);
+        stream.close();
```

Comment:
actually, if something goes wrong in the test, this won't be closed, it would be better if this is inside a try-finally